docs/scos/user/features/202304.0/push-notification-feature-overview.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Push Notification feature overview
+description: The Push Notifications feature lets you receive push notifications
+last_updated: Feb 2, 2023
+template: concept-topic-template
+---
+
+The *Push Notification* feature lets users subscribe to the web push notifications, which are sent from the server to all registered subscriptions.
+
+{% info_block infoBox "Note" %}
+
+The feature can be integrated into the non-Srpyker customer application only. Spryker features do not support push notifications.
+
+{% endinfo_block %}
+
+## Subscribing to notifications
```

Comment:
This part should have the information about generating and storing the public key which is later used to identify the browser to send notifications to

Reply:
I'll try to come up with something. I'll move it out to a separate chapter though